src/components/Schedule/Data.jsx

```diff
@@ -3,86 +3,114 @@ export const startTime = 'August 10, 2021 18:00:00';
 // Date strings are formatted as "YYYY-MM-DDTHH:mm:ss.sss+-hh:mm"
 export const firstEvents = [
```

Comment:
Common practice for constant variables is to use all capitals such as FIRST_EVENTS.